resource/file/file.go

```diff
@@ -105,29 +101,13 @@ func IsMode(f resource.Filer, mode os.FileMode) error {
 // IsOwnedBy checks if the provided file is owned by username user
 // It returns an error if os.Stat returns error
 func IsOwnedBy(f resource.Filer, username string) error {
-	fi, err := f.Info()
-	if err != nil {
-		return nil
-	}
-	uid, err := utils.RoleToID("user", username)
-	if err != nil {
-		return err
-	}
-	return isTrueOrError(fi.Sys().(*syscall.Stat_t).Uid == uint32(uid), fmt.Errorf("%s file is not owned by %s", f, username))
+	return isOwnedBy(f, username)
```

Comment:
Instead doing this, don't you think that it could be helpful to move the methods: `IsOwnedBy` & `IsGrupedInto` to a new interface and "force" implementation of those?